### PR TITLE
New file updated

### PR DESCRIPTION
**Pull request by BhagyashriSSavant**
New folder created on repository Bhagyashri

### CODE REVIEW

File: Bhagyashri/first_assignment_17-july-2019/css/style.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
+
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment_17-july-2019/css/style.css]: Do not add the bhagyashri_firstassignment_17-July**'**19 .
Single quote in branch name this is not good idea to add.

File: Bhagyashri/first_assignment_17-july-2019/css/style.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+body{
+background-color:#71D0C5;
+padding:0 100px;
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment_17-july-2019/css/style.css]: Here it is better that you use padding in percentage. 
As we need to cover the other devices as well.

File: Bhagyashri/first_assignment_17-july-2019/css/style.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+margin:0;
+}
+.first-sect1{
+	   border-top:0;
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment_17-july-2019/css/style.css]: we don't need border to set to zero as it is already zero.

Comment (BhagyashriSSavant) [reply Bhagyashri/first_assignment_17-july-2019/css/style.css]: okay

File: Bhagyashri/first_assignment_17-july-2019/css/style.css (hunk 1 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/first_assignment_17-july-2019/css/style.css]: will take care of it.

**Comment by BhagyashriSSavant** [comment]
done

**Comment by BhagyashriSSavant** [comment]
will take care of it while saving

**Pull request closed without merging**